docs/modules/mssql.md

```diff
@@ -82,6 +82,68 @@ Drops the applicable user from the SQL Server.
 
 Deletes both the database and the SQL Server instance.
 
+##### Examples
+
+###### Kubernetes
+
+The `contrib/k8s/examples/sqldb/sqldb-all-in-one-instance.yaml` can be used to provision one of the plans from the all-in-one `azure-sql` service. This can be done with the following example:
+
+```console
+kubectl create -f ../../contrib/k8s/examples/sqldb/sqldb-all-in-one-instance.yaml
```

Comment:
Would it be possible here (and anywhere else in this doc) to use a URL? It's always annoying that one has to clone a repo just to run some of these commands.

Reply:
Yeah, I think we could do that.